assets/js/modules/analytics-4/index.js

```diff
@@ -196,8 +195,8 @@ export const registerWidgets = ( widgets ) => {
 					return false;
 				}
 
-				const availableAudiences =
-					select( MODULES_ANALYTICS_4 ).getAvailableAudiences();
+				const { availableAudiences } =
+					select( MODULES_ANALYTICS_4 ).getSettings() || {};
```

Comment:
@techanvil @ankitrox It would be nice to include a comment here as to why we specifically don't use the dedicated selector as it isn't obvious.
Alternatively, I wonder if we should consider creating a new selector for that resolver to be attached to instead as it isn't expected that using a setting-getter selector like this would result in an API call?

Reply:
Thanks @aaemnnosttv, I agree it would be preferable to create a new selector for this resolver so we can use `getAvailableAudiences()` without side effects. I've created a new issue for this: https://github.com/google/site-kit-wp/issues/10449